.envrc.template

```diff
@@ -87,3 +87,7 @@ export ERL_FLAGS="+MIscs 2048"
 ## Used for locally building (and eventually testing) Storybook and Snapshot tests
 ## The project token can be found in Chromatic, (Manage > Configure > Project > Project Token)
 # export CHROMATIC_PROJECT_TOKEN=
+
+## Open Route Service
+export OPEN_ROUTE_SERVICE_API_URL=https://api.openrouteservice.org/
```

Comment:
**nitpick:** with the other environment variables in this file we've just been documenting them here and then for information on actual values to use putting them in Notion. I don't think there's any inherent issue with putting this URL in the repo (it's not secret or anything), but I think it would probably be best to be consistent with what we've done so far.